docs/learn/bitesize/bitesize-socialhub.md

```diff
@@ -1,30 +1,32 @@
 ---
-id: bitesize-usecases
-title: Multiplayer Use Cases sample
+id: bitesize-socialhub
+title: Multiplayer Distributed Authority Social Hub sample
 description: Learn about the scenes in the Multiplayer Use Cases sample.
```

Comment:
Oh this should be changed.
I duplicated the Use Cases doc for formatting, but forgot to update these fields.